gammapy/maps/geom.py

```diff
@@ -122,7 +120,10 @@ def find_and_read_bands(hdu, header=None):
             name = re.search("(.+)_MIN", cols[0]).group(1)
         else:
             name = cols[0]
-
+        intp = "INTERP%i" % (i+1)
+        if header is not None:
+            if intp in header:
+                interp = header[intp]
```

Comment:
Add input validation, i.e. raise ValueError or warning or silently skip if an `interp` that Gammapy doesn't understand occurs? Otherwise you put a value in, and it will probably crash or silently misbehave later? Might be useful to have a test for that?

Reply:
`Added a `log.warning()`. Will take default vaules in such cases